src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -446,7 +458,13 @@ protected function getFile($uri, $mimeType)
             $uri = $tempFile;
         }
 
-        return file_get_contents($uri);
+        $file = @file_get_contents($uri);
```

Comment:
this should be handled by the storage class. but it should be fixed in the PR https://github.com/sulu-io/sulu/pull/1139/files where this is refactored.